flang/lib/Lower/ConvertExpr.cpp

```diff
@@ -518,14 +525,13 @@ class ExprLowering {
   fir::ExtendedValue
   genval(const Fortran::evaluate::Extremum<Fortran::evaluate::Type<TC, KIND>>
              &op) {
-    bool isMax = op.ordering == Fortran::evaluate::Ordering::Greater;
     auto lhs = genunbox(op.left());
     auto rhs = genunbox(op.right());
```

Comment:
Will this need to work with arrays at some point?

Reply:
That's likely, I have not rewritten it yet but that would make sens.